frontend/src/fixtures/currentUserFixtures.js

```diff
@@ -86,6 +86,46 @@ const apiCurrentUserFixtures = {
             }
         ]
 
+    },
+    userNoName: {
```

Comment:
It may seem picky, but to avoid confusing future devs, can we call this 
```suggestion
    userNoGivenName: {
```

It's confusing that it's called `noName` when I see
```
           "fullName": "Phillip Conrad",
            familyName": "Conrad",
```

But calling it `noGivenName` makes it much more clear what the purpose is.